File: 9. Palindrome Number/Solution.java
class Solution {
    public boolean isPalindrome(int x) {
        int ori=x;
        int rev=0;
        while(x!=0 && x>=0){
            rev = rev*10 + x%10;
            x = x/10;
        }
        if (ori == rev){
            return true;
        }
        else{
            return false;
        }  
    }
}

File: 9. Palindrome Number/Solution.cpp
class Solution {
public:
    bool isPalindrome(int x) {
        int ori = x;
        long rev = 0;
        while(x>0){
            rev = rev*10 + x%10;
            x = x / 10;
        }
        return (ori == rev);
    }
};